test/internals/plugin.test.js

```diff
@@ -75,7 +75,7 @@ test('checkDecorators should check if the given decorator is present in the inst
     pluginUtils.checkDecorators.call(context, fn)
     t.fail('should throw')
   } catch (err) {
-    t.is(err.message, `The decorator 'plugin' is not present in Request`)
+    t.ok(/The decorator 'plugin' is not present in Request/.test(err.message))
```

Comment:
Why not use `t.match`?

Reply:
Because it was not used in other tests (they used this pattern). If requested, I can change that.

Reply:
I think it would be better to use the framework provided assertion. It will indicate that a test failed because the string does not match the provided regex. Using `t.ok` it will only report that a value was `false` instead of `true`.